src/Form/ModelEditSubmission.tsx

```diff
@@ -20,7 +20,7 @@ const ModelEditSubmission = ({
   onSubmit: any
   activeStep: number
   setActiveStep: Function
-  modelUploading: boolean
+  modelUploading: boolean | undefined
```

Comment:
How are we handling an undefined modelUploading value here? Hard to see in this big PR!